### PR TITLE
drivers/l1-nvram-eeprom: stm32l1 internal EEPROM driver

### PR DESCRIPTION
**Pull request by Cr0s**
Hello everyone,
I'm glad to introduce a simple driver for the STM32L1 Data EEPROM based on the RIOT's NVRAM interface.

Tested on my STM32L151CCU6-based device with testing app located in tests/driver_l1_nvram_eeprom

### CODE REVIEW

**Comment by PeterKietzmann** [comment]
@gebart do you know the stm32l1? Feel free to reassign....

**Comment by gebart** [comment]
@PeterKietzmann I don't have it nor have I worked with it, sorry. I can do a quick check of the NVRAM API parts though. Do you know any maintainers with a board with an STM32L1?

File: drivers/l1-nvram-eeprom/l1-nvram-eeprom.c [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+ * details.
+ */
+
+#include <stdint.h>
+#include <stddef.h>
+
+#include "stm32l1xx.h"
+
+#include "nvram.h"
+#include "xtimer.h"
```
Comment (gebart) [inline drivers/l1-nvram-eeprom/l1-nvram-eeprom.c]: not used anywhere in the code

**Comment by PeterKietzmann** [comment]
I have a nucle-l1 disco board somewhere. Is the eeprom on it? However, I currently don't find the time, probably at the next Hack'n'ACK.

**Comment by Cr0s** [comment]
@PeterKietzmann in case the actual board you have is Nucleo-L152RE, yes, the STM32L152RE on it has a built-in EEPROM.

**Comment by PeterKietzmann** [comment]
It is a STM32L152RET6. The tests seems successful tough I didn't read the code. @gebart you can find the output [here](https://drive.google.com/file/d/0B2ig4jIQ4JbHWGphWmppQjlISDQ/view?usp=sharing)

**Comment by PeterKietzmann** [comment]
@Cr0s did I get it correctly that the driver is special for stm32l1 MCUs? In that case, shouldn’t the file lie somewhere under cpu/stm32l1/ ? I just stumbled upon Murdocks build which (unsurprisingly) failed for several boards that don't know about STM headers and stuff

**Comment by Cr0s** [comment]
Yes, may be it should be located somewhere in `cpu/stm32l1/periph/` instead.

**Comment by smlng** [comment]
@Cr0s still willing to work on this, otherwise we'll try someone to adopt and finish this? As suggested this should likely move into `cpu/stm32l1/periph`.

**Comment by smlng** [comment]
remove milestone for time being

**Pull request closed without merging**